dashboard/src/components/Header/Menu.tsx

```diff
@@ -27,8 +29,26 @@ function Menu({ clusters, appVersion, logout }: IContextSelectorProps) {
   // Control when users click outside
   const ref = useRef(null);
   useOutsideClick(setOpen, [ref], open);
+  const [isDarkModeEnabled, setIsDarkModeEnabled] = useState(
+    localStorage.getItem("theme") === SupportedThemes.dark ? true : false,
+  );
```

Comment:
I first implemented that way, but, since the component (menu) is just handling the `checked` property of the toggle, I thought the state vas should be precisely this one and not the theme.
Especially, if we support more than dark/light themes in the future.

However, I don't have a strong opinion here, as you may prefer.